### PR TITLE
Preview cleaner 131

### PR DESCRIPTION
**Pull request by TrushynAliaksandr**

### CODE REVIEW

File: .buildkite/scripts/github_token.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+set -eo pipefail
+
+set +x
+GITHUB_TOKEN=$(vault read -field=value secret/ci/elastic-docs/docs_preview_cleaner)
+export GITHUB_TOKEN
+set -x
+
+# TODO: Remove this temporary workaround
+# Overwrite known_hosts file with the latest GitHub SSH key
+ssh-keyscan -t rsa github.com > ~/.ssh/known_hosts
```
Comment (nkammah) [inline .buildkite/scripts/github_token.sh:11]: you need a new line at the end of the file

File: .buildkite/scripts/github_token.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (nkammah) [inline .buildkite/scripts/github_token.sh:1]: what's this script about?

File: preview/clean.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ function Cleaner(token, repo, cache_dir, tmp_dir) {
         if (found) {
           acc.push({
             branch: found[1],
-            repo: found[2],
+            repo: found[2].includes('_bk') ? found[2].replace('_bk', '') : found[2],
```
Comment (nkammah) [inline preview/clean.js:40]:
```suggestion
            # Temporary hack during the Buildkite migration that breaks the <repo>_<PR> convention, and introduces the <repo>_BK_<PR> convention - see https://github.com/elastic/docs-projects/issues/134
            repo: found[2].includes('_bk') ? found[2].replace('_bk', '') : found[2],
```

File: .buildkite/scripts/github_token.sh (hunk 2 above)
Comment (TrushynAliaksandr) [reply .buildkite/scripts/github_token.sh]: you are right, forget delete this script after merging

File: preview/clean.js (hunk 3 above)
Comment (TrushynAliaksandr) [reply preview/clean.js:42]: done

**Review by nkammah — APPROVED** [review]

**Pull request merged**